kafka/templates/topics.yaml

```diff
@@ -0,0 +1,19 @@
+{{- range $topic := .Values.kafka.topics }}
+---
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaTopic
+metadata:
+  name: {{ $topic.name }}
+  namespace: {{ $topic.namespace | default $.Release.Namespace }}
```

Comment:
I think the namespace should always be the release namespace unless we come across a future use case for putting the topic in another namespace.  This implies that we support a multi-namespaced deployments in a way that we do not

Reply:
Good idea, this was changed 👍🏼